untitled1/src/main/com/aca/homework/week9/calculator/simple/SimpleCalculator.java

```diff
@@ -0,0 +1,65 @@
+package com.aca.homework.week9.calculator.simple;
+
+import java.util.Scanner;
+
+public class SimpleCalculator {
+    private StringSupplier stringSupplier;
+
+    public SimpleCalculator(StringSupplier stringSupplier) {
+        this.stringSupplier = stringSupplier;
+    }
+
+    public String askAndCalculate() {
+        String expression = stringSupplier.get();
+
+        int lengthOfExpression = expression.length();
+        if (lengthOfExpression != 3) {
+            System.out.println("The user input expression having other format then \' digit operation digit \' ");
+            return "error";
+        }
+
+        char operation = expression.charAt(1);
+        int firstOperand = convertCharToDigit(expression.charAt(0));
+        int secondOperand = convertCharToDigit(expression.charAt(2));
+
+        if (isCharOperation(operation) && firstOperand != -1 && secondOperand != -1) {
+            switch (operation) {
```

Comment:
let's create an OperationType enum, abstract method in it and calculate the result there